app/views/devise/passwords/edit.html.erb

```diff
@@ -0,0 +1,36 @@
+<section class="section">
+  <div class="container">
+    <div class="columns is-centered">
+      <div class="column is-4">
+
+        <h2 class="title is-2">Change your password</h2>
+
+        <%= simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f| %>
+          <%= f.error_notification %>
+
+          <%= f.input :reset_password_token, as: :hidden %>
+          <%= f.full_error :reset_password_token %>
+
+          <div class="field">
+            <div class="control">
+            <%= f.input :password, label: "New password", required: true, autofocus: true, input_html: { class: "input"}, wrapper: false, label_html: { class: "label" }, hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length) %>
```

Comment:
- [ ] at the end of this line theres an If statement, put the if in a helper method that you call here and render this line within the helper method instead of having the if statement directly in the view